.codesandbox/ci.json

```diff
@@ -0,0 +1,7 @@
+{
+  "installCommand": "-v & pnpm i -g pnpm & pnpm i",
+  "buildCommand": "-v & pnpm build",
```

Comment:
`installCommand` and `build` are essentially unnecessary, but the following error will occur if you do not install the latest pnpm in advance.

https://github.com/poteboy/kuma-ui/blob/63a6f80e1eb4e6f36d871cedd7ff1c986dfa01c8/.codesandbox/ci.json

![スクリーンショット 2023-07-13 20 38 21](https://github.com/poteboy/kuma-ui/assets/12913947/56f05a39-2886-444d-8ef0-6e95f2ccfd90)

I am using a somewhat unconventional method because when using `installCommand`, the command is executed using npm instead of pnpm, for some reason.